HHS/Head-Start-TTADP
# Conditional display of the "New Activity Report" button based on permissions

Description of change
Adds a conditional display both in the empty and non-empty state based on user's permission. The button will be displayed if the user has READ_WRITE_ACTIVITY_REPORTS

How to test

Pull changes
Simulate starting the app as a user who doesn't have read/write permissions
(if using Harry Potter, e.g. - the permissions will have to be first deleted either via the app or by
DELETE FROM public."Permissions"
	WHERE id in (5,7);
Note that there won't be a button displayed allowing to create a new report
Restart the server simulating a user that does have permissions
Note that the button is now displayed
Issue(s)

**Issue(s)**
* closes https://github.com/HHS/Head-Start-TTADP/issues/153

**Checklist**

- [na] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
